drizzle/0000_flowery_union_jack.sql

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE IF NOT EXISTS "code_group_mappings" (
```

Comment:
was this file autogenerated or manually written? It seems to me a bit convoluted to create the tables and then alter them to add foreign key constraints instead of having those constraints as part of the table definition.

Reply:
It was autogenerated when running the commend `npx drizzle-kit generate`